packages/meditrak-server/src/tests/dhis/DhisChangeDetailGenerator.test.js

```diff
@@ -13,13 +13,20 @@ import {
   TONGA_SURVEY_RESPONSE,
 } from './DhisChangeDetailGenerator.fixtures';
 
-const buildChange = (type, record) => ({
-  record_type: type,
-  record: {
-    id: generateTestId(),
-    ...record,
-  },
-});
+const buildChange = (type, record) => {
+  const recordId = generateTestId();
+  return {
+    record_type: type,
+    new_record: {
+      id: recordId,
+      ...record,
+    },
+    old_record: {
+      id: recordId,
+      ...record,
+    },
```

Comment:
While the DhisChangeDetailGenerator _does_ respond to the `publishRecordUpdates` fairly often, I've taken your suggestion and made `old_record` null, so that these are testing the situation of a newly created record.